topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.md

```diff
@@ -33,9 +33,9 @@ contributions:
 
 # Introduction
 
-Food contamination with pathogens are a major burden on our society. In the year 2019, foodborne pathogens caused 137 diseases in Germany [(BVL 2019)](https://www.bvl.bund.de/SharedDocs/Berichte/10_BELA_lebensmittelbed_Krankheitsausbruechen_Dtl/Jahresbericht2019.pdf?__blob=publicationFile&v=4). Globally, they affect an estimated 600 million people a year and impact socioeconomic development at different levels. These outbreaks are mainly due to _Salmonella spp._ followed by _Campylobacter spp._ and Noroviruses.
+Food contamination with pathogens are a major burden on our society. In the year 2019, foodborne pathogens caused 137 diseases in Germany [(BVL 2019)](https://www.bvl.bund.de/SharedDocs/Berichte/10_BELA_lebensmittelbed_Krankheitsausbruechen_Dtl/Jahresbericht2019.pdf?__blob=publicationFile&v=4). Globally, they affect an estimated 600 million people a year and impact socioeconomic development at different levels. These outbreaks are mainly due to _Salmonella spp._ followed by _Campylobacter spp._ and Noroviruses, as studied by the [__Food safety - World Health Organization (WHO)__](https://www.who.int/news-room/fact-sheets/detail/food-safety) in 2022.
```

Comment:
Couple of issues with this paragraph:
- The WHO link leads to a page that was *updated* in 2022, but if you read the text, the number 600 million is from a *2015* report. That report (https://www.who.int/publications/i/item/9789241565165) makes the better link I suppose.
- It's possible that the pathogen importance order you're giving is contained somewhere in that long report, but it's not given on the page you're currently linking to
- the German (BVL2019) report is also not summarized correctly: 137 is the number of hospitalisations (not diseases) caused by disease outbreaks for which there was strong evidence that they were caused by a foodborne pathogen.